### ISSUE TITLE
Hello Keijiro, there are multiple issues on unity5b22

### ISSUE THREAD

**Issue by pointcache** [keijiro/KinoFog#2]
I dont really understand the reason, but if you drop it in scene and press "fade to skybox" you will see multiple errors.

**Comment by keijiro** [comment]
What errors did you get? Could you copy-and-paste the error messages here?

**Comment by pointcache** [comment]
http://i.imgur.com/n4bUmiu.png
these, im trying again with 5.4f2 and getting these, i guess some minor name changes should be done, im afraid im too bad with shaders to fix it, but ill try

**Comment by pointcache** [comment]
Huh it seems that when i try to use unity 5 procedural skybox it doesnt work :| any chance you will add support for procedural one?

**Comment by pointcache** [comment]
yep its 100% the cause if i switch to old cubemap shader all works, but not in procedural.

**Comment by keijiro** [comment]
No plan at the moment but worth thinking. Please keep this issue opened. I'll think it again later.

**Comment by pointcache** [comment]
i see that procedural doesnt have a tex slot that you use, it is possible to make a "snapshot" (i had old script somewhere that made reflection cubemaps in arbitrary level parts) of the procedural skybox and insert it as cubemap. Its a hack and wont work on runtime, but i dont care for my use, i could drop the thing as optional for the likes of me.

**Comment by GeorgeBroussard** [comment]
Keiliro,

Just a note that I enjoy KinoFog, but would really love it if it could work on procedural skyboxes as well. Had difficulty finding a simple solution to matching fog to procedural skyboxes at the horizon.

**Comment by yaakovyitzchak** [comment]
@GeorgeBroussard Im also having issues with this any ideas? https://forum.unity.com/threads/fog-clipping.1127831/
